DFC.ServiceTaxonomy.Neo4j/Services/NeoGraphDatabase.cs

```diff
@@ -29,17 +30,17 @@ public class NeoGraphDatabase : IGraphDatabase, IDisposable
             // It is considerably cheap to create new sessions and transactions, as sessions and transactions do not create new connections as long as there are free connections available in the connection pool.
             //  driver is thread-safe, while the session or the transaction is not thread-safe.
             //todo: add configuration/settings menu item/page so user can enter this
-            var neo4jConfiguration = neo4jConfigurationOptions.CurrentValue;
-
-            _driver = GraphDatabase.Driver(
-                neo4jConfiguration.Endpoint.Uri,
-                AuthTokens.Basic(neo4jConfiguration.Endpoint.Username, neo4jConfiguration.Endpoint.Password),
-                o => o.WithLogger(neoLogger));
+            _drivers = driverBuilder.Build();
+            _currentDriver = 0;
```

Comment:
Don't think this is necessary.

Reply:
The class is a singleton so current driver is set to 0, and is set back to 0 in the next driver method

Reply:
I believe fields uninitialised in the constructor are set to their default values (i.e. 0).